src/index.html

```diff
@@ -22,6 +25,395 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav_list">
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link is-active"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link"
+              href="#Samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link"
+              href="#Smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              data-qa="nav-hover"
+              class="header__nav_link"
+              href="#Laptops&Computers"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link"
+              href="#Gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link"
+              href="#Tablets"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link"
+              href="#Photo"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav_item">
+            <a
+              class="header__nav_link"
+              href="#Video"
+            >
+              Video
+            </a>
+          </li>
```

Comment:
The BEM naming convention should be consistently followed. Please update all occurrences of `header__nav_item` to `header__nav-item` and `header__nav_link` to `header__nav-link`.